R/object-builder.R

```diff
@@ -27,9 +27,9 @@ build_dfm <- function(x, features,
                       docvars = data.frame(), meta = list(), 
                       class = "dfm", ...) {
     result <- new(class,
-        as(x, "dgCMatrix"),
-        docvars = docvars,
-        meta = make_meta("dfm", inherit = meta, ...)
+                  as(x, "CsparseMatrix"),
```

Comment:
If you add `as(as(as(x, "CsparseMatrix"), "generalMatrix"), "dMatrix")` in `build_fcm()`, let's do the same here too. This may contradict my earlier comment, but I think it we are better off if the two object builders work consistently.

Reply:
You're right, better to make sure here, since we want only a dgCMatrix returned, and not some other version of CsparseMatrix. It doesn't hurt to be sure.